### PR TITLE
Fix show password button on the login screen (Lombiq Technologies: OCORE-172)

### PR DESCRIPTION
**Pull request by Piedone**
I checked for similar issues but this seems to be the only one.

This is a regression post-1.8.

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Users/Views/LoginFormCredentials.cshtml [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 <script at="Foot">
     togglePassword = document.querySelector('#togglePassword');
-    password = document.querySelector('#Password');
+    password = document.querySelector('#@Html.IdFor(m => m.Password)');
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Users/Views/LoginFormCredentials.cshtml:26]: Please add the directive `let` before password and togglePassword. These are local variables

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Users/Views/LoginFormCredentials.cshtml:26]: What are the difference except the intellisense, both have "#Password" value

Comment (Piedone) [reply src/OrchardCore.Modules/OrchardCore.Users/Views/LoginFormCredentials.cshtml:26]: The old hard-coded ID that the script depended on was `Password` but the new one generated by `asp-for="Password"` is `LoginForm_Password`. This is why the script was broken.

**Review by MikeAlhayek — APPROVED** [review]
It may not be a bad idea to also use model binding to create the toggler ID too this way it's consistent. Either way, I approve feel free to merge or update then merge

**Review by hishamco — APPROVED** [review]

**Comment by Piedone** [comment]
> It may not be a bad idea to also use model binding to create the toggler ID too this way it's consistent. Either way, I approve feel free to merge or update then merge

I think that can remain; It's a single-use element that doesn't have a corresponding member in the view model.

Thanks both!

**Pull request merged**